app/assets/stylesheets/criteria.scss

```diff
@@ -80,11 +80,17 @@
     border-left: 1px solid $grey;
     display: table-cell;
     padding-left: 1em;
+    width: 100%;
 
     .rubric_level_header {
       margin-bottom: 0.5em;
     }
 
+    .rubric_level_title {
+      font-weight: bold;
+      margin: 0 3px 5px 0;
```

Comment:
when I run rubocop on this file, there are no offenses. Is this something that I should change? If so, what exactly does this change mean?

Reply:
This is a `.scss` file, not Ruby, so Hound uses a different linter. You can ignore this for now, since it's likely this selector will be removed as you implement the changes we discussed at our meeting.